assets/javascripts/initializers/code-bytes.js

```diff
@@ -145,7 +145,12 @@ function initializeCodeByte(api) {
       const params = [];
       params.push(`lang=${language}`);
       params.push(`text=${Base64.encodeURI(text)}`);
-      params.push(`mode=${isPreview ? 'compose' : 'view'}`);
+
+      params.push(`client-name=forum`);
+      params.push(`page=${Base64.encodeURI(postUrl)}`);
```

Comment:
Actually `postUrl` is a bit of a misnomer as it doesn't include the base URL, so we'll want to concatenate that.